tests/data/ext/sample-status-invalid-json-format.json

```diff
@@ -0,0 +1,36 @@
+[
```

Comment:
could you add a comment explaining what makes this file invalid?

you could, for example, add a dummy property to the json:

> "_comment": "This is an invalid status file, it's missing a brace at line XX"